### ISSUE TITLE
Text objects are offset on the Y axis (Linux; Firefox, Brave)

### ISSUE THREAD

**Issue by theblacknight** [photonstorm/phaser#5710]
## Version

* Phaser Version: 3.54.0

* Operating system: Linux, Fedora 32

* Browser: Firefox, Brave

## Description

Text objects are not being rendered in the correct position. 

The link below shows this happening in the Phaser 3 examples page.

A text object is added to the scene added at Y = 100, but the text is rendered at closer to Y = 500.

I first had this issue several months ago on Firefox. I didn't want to get side tracked so I tried Brave, and everything worked as expected. However Brave is now displaying this behavior too.

I have tried on OSX with Firefox and Chrome and it works fine there, so looks like a linux/browser issue. Interested to know if anyone else can repro.

Any help would be greatly appreciated.

## Example Test Code

This issue is impacting Phaser examples: https://www.phaser.io/examples/v3/view/components/data/store-scene-data

**Comment by theblacknight** [comment]
A little more information on this.

It looks like the issue is in calculating some sort of padding above the text. The size of that padding also scales with the font size. 

It also affects each line of multi-text.

When looking at the data, the bounds of the object are way too large. 

Again, I've only confirmed in Brave and Firefox on Fedora (Linux) 32 & 33. 

Anyway, I decided to use bitmapText objects as a work around. Less flexibility, but they're working fine and are enough for my needs.

Cheers,

- Kev
